src/main/java/whobot/main/UI.java

```diff
@@ -0,0 +1,99 @@
+package whobot.main;
+
+import java.util.Scanner;
+
+public class UI {
+
+    /** Colors used to Display Text */
+    public static final String COLOR_RESET = "\u001B[0m";
+    public static final String COLOR_BLUE = "\u001B[34m";
+    public static final String COLOR_CYAN = "\u001B[36m";
+    public static final String COLOR_PURPLE = "\u001B[35m";
+    public static final String COLOR_RED = "\u001B[31m";
```

Comment:
Very good and clear naming for these constants (colour, commands, etc.). Readability greatly enhanced!